src/index.js

```diff
@@ -2,16 +2,6 @@ let i = 0;
 const txt = 'Welcome to our online bootcamp'; /* The text */
 const speed = 100; /* The speed/duration of the effect in milliseconds */
 
-let ghpages = require('gh-pages');
-
-ghpages.publish(
-  '.',
-  {
-    repo: 'git@github.com:bekjon-me/capstone-project.git',
-  },
-  callback
-);
-
 function typeWriter() {
   if (i < txt.length) {
     document.getElementById('typingTitle').innerHTML += txt.charAt(i);
```

Comment:
- [ ] The list of speakers used to generate the HTML should be assigned to a variable as an array or object. you can use an array to store objects (speakers) having all the information needed to generate the dynamic element, at least the name, role, description, and image URL about the speaker.
Note: About the array of objects you can check this [array of object](https://www.freecodecamp.org/news/javascript-array-of-objects-tutorial-how-to-create-update-and-loop-through-objects-using-js-array-methods/)
- [ ] Also include a function that iterates through the speakers list on window load